fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2236,4 +2236,17 @@ public class Config extends ConfigBase {
 
     @ConfField
     public static boolean enable_execute_script_on_frontend = true;
+
+
+    /**
+     * The default scheduler interval for alter jobs.
+     */
+    @ConfField(mutable = true)
+    public static int default_alter_scheduler_interval_millisecond = 10000;
```

Comment:
After making it configurable, we can remove `default_` prefix?

Reply:
Done